src/index.ts

```diff
@@ -353,6 +353,20 @@ export function refreshConsentList(clientAddress: string) {
   XMTPModule.refreshConsentList(clientAddress)
 }
 
+export async function consentList(
+  clientAddress: string
+): Promise<Map<string, 'allowed' | 'denied' | 'unknown'>> {
```

Comment:
@alexrisch I'm sure theres a better typescript way to do these enum like strings?

Reply:
Agree with @alexrisch , your implementation is fine. Marginal improvement IMO would be to export a type that you can reuse like in this diff => https://github.com/xmtp/xmtp-react-native/compare/np/bump-pod-for-new-pppp...np/bump-pod-for-new-pppp-union-type?expand=1

Exporting an enum like in @alexrisch 's link is the most flexible if we want to err on the side of developer choice, but I think the slightly more opinionated string literal union type is the default here.